wallet/src/de/schildbach/wallet/service/CoinJoinService.kt

```diff
@@ -66,9 +66,10 @@ enum class CoinJoinMode {
 }
 
 interface CoinJoinService {
+    var mode: CoinJoinMode
+    val mixingStatus: MixingStatus
+
     fun needsToMix(amount: Coin): Boolean
-    fun getMode(): CoinJoinMode
-    fun setMode(mode: CoinJoinMode)
```

Comment:
I think in Kotlin we don't need dedicated setter/getter methods

Reply:
Yeah, this was probably due to thinking in Java.